### PR TITLE
Update use case

### LINKED ISSUE #208: Update Use Cases

### PR DESCRIPTION
**Pull request by yu-sutong**
close #208

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -290,14 +290,15 @@ _{Explain here how the data archiving feature will be implemented}_
...
-| `* * *`  | user                                        | mark orders as completed                                     | track which orders are still outstanding                             |
-| `* * *`  | user                                        | delete an order                                              | remove orders that have been canceled                                |
-| `* * *`  | user with many contacts in the address book | delete a customer from my address book                       | remove clients who I no longer need to be in contact with            |
+| `* * *`  | user                                        | add orders related to a customer                             | keep track of deliveries to be made                                   |
+| `* * *`  | user                                        | add details to orders, such as delivery date and item        | know what needs to be delivered and when                              |
+| `* * *`  | user                                        | mark orders as completed                                     | track which orders are still outstanding                              |
```
Comment (FooChao) [inline docs/DeveloperGuide.md:298]: isn't this kx part?

Comment (yu-sutong) [reply docs/DeveloperGuide.md:298]: oh

Comment (yu-sutong) [reply docs/DeveloperGuide.md:298]: deleted

**Review by FooChao — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -449,11 +449,28 @@ Use case ends.
...
+* **Use Case**: UC08 - Filter an Order List by Order Status
+* **Guarantees**:
+    * Orders with specified status made by specific customer will be displayed as a list, if input parameters are valid.
+
+**MSS**
+
+1.  User wants to filter an order list.
+2.  User chooses to <ins>view all orders of a specified customer from contacts list (UC04)</ins>.
+3.  User filters the order list of the customer by the status keyword.
+4.  SellSavvy displays the orders with specified status in a list in GUI.
```
Comment (FooChao) [inline docs/DeveloperGuide.md:470]: i not sure but 'with the specified status'?

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -473,6 +490,7 @@ Use case ends.
 * **Mainstream OS**: Windows, Linux, Unix, MacOS
 * **Order**: Agreement made by customers with user on delivery of product
 * **Private contact detail**: A contact detail that is not meant to be shared with others
+* **Status**: The current fulfilment condition of an order, such as completed or pending.
```
Comment (kaixuan477) [inline docs/DeveloperGuide.md:493]:
```suggestion
* **Status**: The current fulfilment condition of the delivery of an order, namely completed or pending.
```

Comment (kaixuan477) [reply docs/DeveloperGuide.md:493]: Negligible, although Im not sure if "condition" is the right word, but not much difference I guess.

**Review by kaixuan477 — APPROVED** [review]
Looks good.

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -449,11 +449,28 @@ Use case ends.
 
 1.  User wants to delete an order.
 2.  User chooses to <ins>view all orders of a specified customer from contacts list (UC04)</ins>.
-3.  User deletes the customer by the customer name and the order index.
+3.  User deletes the order by the customer name and the order index.
 4.  SellSavvy updates the status of the order and indicates that the action is successful.
 
 Use case ends.
 
+**Use case 8: Filter order list by order status**
```
Comment (FooChao) [inline docs/DeveloperGuide.md:457]: image

i not sure but can we move it above? to below listorder then those below we can change to chooses to view all orders or filtered orders. can someone help on this? :(

Comment (FooChao) [reply docs/DeveloperGuide.md:457]: Nvm u can ignore this first not ipt

**Pull request merged**